### DISCUSSION TITLE
how to treat zero cost channels and attribution hints

### DISCUSSION THREAD

**Discussion by chanansh** [pymc-labs/pymc-marketing#371] [Q&A]
I have a data frame which looks like this:
date | channel name | cost | leads

Some of the channels' cost is zero. The number of leads per channel is based on a simple attribution like first or last touch.
In MMM usually one just counts the total outcome, e.g. `df.groupby('date').leads.sum()` however, I feel some information is lost. Moreover, how should I treat zero-cost channels? The ROI is infinite.

**Comment by ricardoV94** [comment]
How do you modulate the "free channels"?. For passive channels I think they are usually just modeled as "background" effects.
